### ISSUE TITLE
Wrong aspect ration on CPS1 games

### ISSUE THREAD

**Issue by rereprep** [crazii/fba4dstwo#2]
I only tried on Final Fight but i assume it is true for other CPS games too. CPS hardware has 384 224 (12:7) resolution. But it is meant to be displayed on 4:3 aspect ratio. Pixels arent square. Both MAME and FBA4DSTWO displays it in  (12:7) aspect ratio on DS where it should display in 4:3 aspect ratio with 256 x 192 resolution. This causes stretched picture and black bars on upper and lower parts of the screen.

**Comment by crazii** [comment]
@rereprep 
Yes, the "native" resolution of CPS1 is 12:7, but its ought to be shown on a 4:3 display.

the current implementation is a simple scaling (from 12:7 to 12:7 ),  which need to be fixed by using a proper stretching method I guess.
